homeassistant/components/sql/config_flow.py

```diff
@@ -68,6 +56,74 @@ def validate_query(db_url: str, query: str, column: str) -> bool:
     return True
 
 
+def _generate_schema_config(current_data: dict[str, Any]) -> vol.Schema:
+    """Generate schema for config flow."""
+    return vol.Schema(
+        {
+            vol.Required(
+                CONF_NAME,
+                description={
+                    "suggested_value": current_data.get(CONF_NAME, "Select SQL Query")
+                },
+            ): selector.TextSelector(),
+            vol.Optional(
```

Comment:
You can remove this and use .extend on the result of the other function you added so you don't have duplicate code